src/script/nodes/dialogue/dialogue_message.cpp

```diff
@@ -57,7 +57,7 @@ class OScriptNodeDialogueMessageInstance : public OScriptNodeInstance
             return selection == -1 ? 0 : selection;
         }
 
-        Variant scene_file = p_context.get_input(2);
+        Variant scene_file = p_context.get_input(3);
```

Comment:
Unfortunately, this change won't be backward compatible. You can't change the pin order or indices without there being issues. In short, this means that any older orchestrations won't connect their pins properly.

You may need to take a look at `_upgrade` or `reallocate_pins_during_reconstruction` to deal with correctly handling the upgrade for existing dialogue message nodes in older orchestrations.